src/scripts/populateBoard.js

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
- [ ] [OPTIONAL] What could you return from here to make it testable? Try to think also what you can test in the function that does not necessarily involve the returned value, the arguments, side effects, etc.